xml/depl_maintenance.xml

```diff
@@ -508,6 +508,258 @@
+      </para>
+<screen>psql2mysql.py --batch /etc/pg2mysql/databases.yaml migrate</screen>
+     </step>
+     <step>
+      <para>
+       In the &crow; &wi;, switch to the <guimenu>Database</guimenu>
+       &barcl;. Enable the raw view and set the value of
+       <literal>sql_engine</literal> to <literal>mysql</literal>. Apply the
+       &barcl;. After this step, &ostack; services should be running again and
+       reconfigured to use the &mariadb; database back end.
+      </para>
+     </step>
+     <step>
+      <para>
+        To prevent the &postgres;-related chef code from running, unassign the values
+	from <literal>database-server</literal> role in the <guimenu>Database</guimenu>
+       &barcl;, and apply the &barcl;.
+      </para>
+     </step>
+     <step>
+      <para>
+       Start chef-client services on the nodes again.
+      </para>
+     </step>
+     <step>
+      <para>
+       Stop &postgres; on the &dbnode;s. Uninstall &postgres; packages.
+      </para>
+      <substeps>
+       <step>
+	<para>
+	 To stop the <literal>postgresql</literal> service, run the following
+	 command on one cluster node:
+	</para>
+<screen>crm resource stop postgresql
+crm resource stop fs-postgresql
+crm resource stop drbd-postgresql</screen>
+        <para>
+	 Run the last command only if the previous setup used DRBD.
+	</para>
+       </step>
+       <step>
+	<para>
+	 Remove the packages on all cluster nodes:
+	</para>
+	<screen>zypper rm postgresql94 postgresql94-server</screen>
+       </step>
+       <step>
+	<para>
+	 If you choose not to upgrade to &productname; 8 right away, delete unused pacemaker resource from one cluster node:
+	</para>
+<screen>crm conf delete drbd-postgresql
+crm conf delete fs-postgresql
+crm conf delete postgresql</screen>
+       </step>
+      </substeps>
+     </step>
+     <step>
+      <para>
+       If DRBD is not used as a backend for &rabbit;, it is possible to remove it at this point, using the following command:
+      </para>
+<screen>&prompt.sudo;zypper rm drbd drbd-utils</screen>
+      <para>
+       You can then reclaim the disk space sued by &crow; for DRBD. To do this, edit the node data using <systemitem>knife</systemitem>:
+      </para>
+<screen>&prompt.user;knife node edit -a <replaceable>DRBD_NODE</replaceable></screen>
+      <para>
+        Search for <literal>claimed_disks</literal> and remove the entry with owner set to <literal>LVM_DRBD</literal>.
+      </para>
+      <para>
+       Otherwise, skip this step until after the full upgrade is done, since the &rabbit;
+       setup will be automatically switched from DRBD during the upgrade procedure.
+      </para>
+     </step>
+    </procedure>
+   </sect3>
+   <sect3 xml:id="sec.postgre-mariadb.upgrade.scenario2">
+    <title>HA Cluster with 2 &contrnode;s</title>
+    <procedure>
+     <step>
+     <para>
+     Extend the 2-node cluster with additional node that without any role assigned to
+     it.
```

Comment:
Make sure the new node has enough memory to serve as a controller node.

Reply:
Fixed.